backend-rust/rustfmt.toml

```diff
@@ -0,0 +1,15 @@
+edition = "2021"
```

Comment:
I know we use this non-standard formatting in other projects and you have been unwilling to revert to the standard in those existing projects - but could we at least use the standard formatting for new projects?

Reply:
IMO this should just use the formatting like every other project we have (which is actually not this config, I'll fix that).
It is not because I prefer the config, and I was not the one who defined it either, but it would be consistent.
Also, if we change it for this project, people would have to change editor configs between this and other projects.

Reply:
With that approach, the "cost" of changing to the standard format will only increase and it will only get more and more cumbersome to change the format to the standard. I think it would be better to do a gradual approach where we slowly move towards the standard, and part of that would be new projects using the standard.

Reply:
IMO It is already too cumbersome compared to the value gained from doing it.
Also, it needs to be a team decision to change it, and not something we just randomly do because of preference.

Reply:
I did not write random preference. I was referring to random changes, we need to align people before doing something like this.
"This situation" is not a problem. You just prefer something else, which is fine. It can be discussed whether we want to spend the time it takes to change this in the team, but should not be something we just start doing before it has been discussed.

> The whole point of sticking to the usual standard that is used in all other open source projects

Seems that a lot of open source projects have some rustfmt setup: servo and rust-analyzer even the rust compiler and rustfmt itself have one.